azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/Usage.java

```diff
@@ -7,14 +7,13 @@
 package com.microsoft.azure.management.storage;
 
 import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
-import com.microsoft.azure.management.storage.implementation.api.UsageInner;
 import com.microsoft.azure.management.storage.implementation.api.UsageName;
 import com.microsoft.azure.management.storage.implementation.api.UsageUnit;
 
 /**
  * An immutable client-side representation of an Azure storage resource usage info object.
  */
-public interface Usage extends Wrapper<UsageInner> {
+public interface Usage extends Wrapper<com.microsoft.azure.management.storage.implementation.api.Usage> {
```

Comment:
What's happening is they moved their list into an object like `StorageAccountUsageListResult` and `Usage` is no longer a top level.